en-US/Grammar.xml

```diff
@@ -416,9 +454,189 @@
+				<colspec colname="c1"></colspec>
+				 <colspec colname="c2"></colspec>
+				 <thead>
+					<row>
+						<entry> Example </entry>
+						 <entry> Improvement </entry>
+
+					</row>
+
+				</thead>
+				 <tbody>
+					<row>
+						<entry> Divide the storage devices into smaller chunks to create a partition. </entry>
+						 <entry> To create a partition, divide the storage devices into smaller chunks. </entry>
+
+					</row>
+
+				</tbody>
+
+			</tgroup>
+
+		</table> -->
+
+<!-- Added section about "either, or" -->
+		 <formalpara id="either-or">
+			<title>The Either-Or Construction</title>
+			 <para>
+				Use the "either x ... or y" construction only to refer to a choice between two options, not for three or more options.
+			</para>
+
+		</formalpara>
+		 <table>
+			<title></title>
+			 <tgroup cols="2" colsep="1" rowsep="1">
+				<colspec colname="c1"></colspec>
+				 <colspec colname="c2"></colspec>
+				 <thead>
+					<row>
+						<entry> Example </entry>
+						 <entry> Improvement </entry>
+
+					</row>
+
+				</thead>
+				 <tbody>
+					<row>
+						<entry> Scale up by adding more resources, with either more memory, CPUs, or disk space. </entry>
+						 <entry> Scale up by adding more resources, such as memory, CPUs, or disk space. </entry>
+
+					</row>
+
+				</tbody>
+
+			</tgroup>
+
+		</table>
+
+<!-- Added section about "following" -->
+		 <formalpara id="following-noun">
+			<title>Use of Following</title>
+			 <para>
+				Use "following" with a noun.
+				In a quiz question, "the following" is often superfluous.
+			</para>
+
+		</formalpara>
+		 <table>
+			<title></title>
+			 <tgroup cols="2" colsep="1" rowsep="1">
+				<colspec colname="c1"></colspec>
+				 <colspec colname="c2"></colspec>
+				 <thead>
+					<row>
+						<entry> Example </entry>
+						 <entry> Improvement </entry>
+
+					</row>
+
+				</thead>
+				 <tbody>
+					<row>
+						<entry> Non-privileged users can use the role to configure the following: </entry>
+						 <entry> Non-privileged users can use the role to configure the following interfaces: </entry>
+
+					</row>
+					<row>
+						<entry> Which two of the following statements describe the benefits of Linux? </entry>
+						 <entry> Which two statements describe the benefits of Linux? </entry>
+
+					</row>
+
+				</tbody>
+
+			</tgroup>
+
+		</table>
+
+<!-- Added section about "if, then" -->
+		 <formalpara id="if-then">
+			<title>The If-Then Construction</title>
+			 <para>
+				Generally, follow an "if" statement with a "then" statement.
+			</para>
+
+		</formalpara>
+		 <table>
+			<title></title>
+			 <tgroup cols="2" colsep="1" rowsep="1">
+				<colspec colname="c1"></colspec>
+				 <colspec colname="c2"></colspec>
+				 <thead>
+					<row>
+						<entry> Example </entry>
+						 <entry> Improvement </entry>
+
+					</row>
+
+				</thead>
+				 <tbody>
+					<row>
+						<entry> If one service is started, the other is started as well. </entry>
```

Comment:
I might use "also" instead of "as well", but not critical.

Reply:
Fixed